### ISSUE TITLE
Plugin seems to not reconnect when restarting Galaxy

### ISSUE THREAD

**Issue by micpp42** [tauqua/gog-galaxy-itch.io#23]
I can connect and pull my library in fine, but when I restart Galaxy, although I still see all my games showing in the list the plugin has gone back to Disconnected, and I have to log in again

**Comment by tauqua** [comment]
If you've previously used a different Itch.io plugin, this may be happening due to a Galaxy issue. If you feel comfortable, you can try to edit Galaxy's local database to fix this issue. I've documented the process that has worked for me on this project's [wiki](https://github.com/tauqua/gog-galaxy-itch.io/wiki/Unique-Constraint-Failed-Issue) but I make no guarantees it will work.

You should also double check that you're running the most recent version of the plugin (v0.0.5)

**Comment by LukasThyWalls** [comment]
Hello.

I think i'm in the same issue because i used before Ertego's itch.io integration, and every time i enter Galaxy i see this integration disconnected.

I try the wiki tips, but i can't do nothing. First, it shows as Platform ID 13, not 12, but i suppose something changed in this time since you posted, because the UIDs are correct. However, if i try to delete the registry of the other UID (not the UID from this integration) shows me an error. Maybe is an issue of how to use DB Browser (i never use it before), but Galaxy is not on, and it should let me modify the DB...
![GOG_UniqueConstraintFailedIssue_NotWorking](https://user-images.githubusercontent.com/1467366/112148246-f20e7880-8bdd-11eb-844b-74a58a69ea1a.png)

**Comment by BleakBluets** [comment]
If anyone else is having the "FOREIGN KEY constraint failed" issue when trying to delete an old plugin record from the `Plugins` table of the galaxy-2.0.db database, this happens when a record is still being referenced in another table. These references in other tables depend on this source record existing, so it's protected from being deleted. To fix this, all other reference to that record must be deleted before deleting the record itself. The only other reference to this record I could find is in the `UserPlugins` table.

To use the previous commenter's data as an example, the record which must be deleted is in the `Plugins` table and has an `id` of `20`, which is the primary key that identifies this record as a reference in other tables. When trying to delete this record, a foreign key constraint error is encountered because it still has a reference elsewhere in the database. To fix the issue in this case, go to the `UserPlugins` table and delete the record with the `pluginId` of `20`, which is the foreign key which references our original record. Then, as long as no other references to it exist, the original record in the `Plugins` table can be deleted.

In short, 
Delete the record in the table `UserPlugins` which has a `pluginId` that matches the `id` of the record you want to delete in the `Plugins` table. Once the record in the `UserPlugin` table is deleted, then you can delete the original record in the `Plugins` table.

No reinstalling required.
